graphql/schema/types/config.graphql

```diff
@@ -454,7 +454,6 @@ input ServiceFlagsInput {
   degradedModeDisabled: Boolean
   elasticIPsDisabled: Boolean
   releaseModeDisabled: Boolean
-  adminParameterStoreDisabled: Boolean
```

Comment:
Does the UI (old or new) rely on this at all? Does Aldo's admin page need to be changed for example?

Reply:
it did, i made a change to make them all be optional
and then also deleted them from the UI first in a different PR